### ISSUE TITLE
BC22 - Users created from Setup-BcContainerTestUsers does not have any permission sets

### ISSUE THREAD

**Issue by jwikman** [microsoft/navcontainerhelper#2951]
**Describe the issue**
We use `Setup-BcContainerTestUsers` to create test users with different licenses and permissions.
In NextMajor (BC22) containers, there are no User Groups since (I guess) the User Group feature is deprecated.
`Setup-BcContainerTestUsers` seems to use the User Groups to add permissions to the test users, and since there are no User Groups the users does not get any permissions.

**Scripts used to create container and cause the issue**
```
Setup-BcContainerTestUsers -containerName $containerName -credential $credential -Password $TestUsersPassword
```

**Full output of scripts**
```
Downloading C:\Users\nabjohwik\AppData\Local\Temp\267f3091-3a2d-41bd-ba17-333c3c547195
Using WebClient
VERBOSE: Preparing to expand...
VERBOSE: Created 'C:\Users\nabjohwik\AppData\Local\Temp\9aaaf62d-bc70-4c75-89bb-d4144ea61f8f\Microsoft_CreateTestUsers_16.0.7.0.app'.
Publishing C:\ProgramData\BcContainerHelper\Extensions\nextmajor\e0477fb6-7024-434d-acba-3069485cf78c\Microsoft_CreateTestUsers_16.0.7.0.app
Synchronizing CreateTestUsers on tenant default
Installing CreateTestUsers on tenant default
App Microsoft_CreateTestUsers_16.0.7.0.app successfully published
Invoke GET on https://172.17.254.24:7048/BC/api/v1.0/companies?$filter=name%20eq%20%27CRONUS%20SE%27&tenant=default
Invoke POST on https://172.17.254.24:7048/BC/api/Microsoft/Setup/beta/companies(74629193-f9c9-ed11-94ca-000d3abcda6d)/testUsers?tenant=default
Uninstalling CreateTestUsers from tenant default
Unpublishing CreateTestUsers
App successfully unpublished
```

**Screenshots**
N/A

**Additional context**
- does it happen all the time - Yes, for all NextMajor containers
- did it use to work - Yes, for BC21
- BCArtifactUrl: https://bcpublicpreview.azureedge.net/sandbox/22.0.54157.54644/se

**Comment by freddydk** [comment]
I thought this was due to the new User Group functionality, which is turned on by default, but even if I add 

```
-featureKeys @{ "HideLegacyUserGroups" = "None" }
```

to New-BcContainer, then it still seems to be a problem.
I have sent the TestUsers app to somebody in the app team to get it compatible with 22

**Comment by jwikman** [comment]
I think the test data in the database has changed. I believe there are no User Groups there, independent of if you are using User Groups or not.

If that's case, then the app needs to be redesigned and permissions added directly on users (or some security group mockup thing). :)
The app needs to be updated to support that User Groups are completely removed in an upcoming major anyways...

**Comment by freddydk** [comment]
It has been redesigned with the change submitted at the same time as you replied:-)

**Comment by jwikman** [comment]
Haha, nice! :)

**Comment by freddydk** [comment]
Shipped in 5.0.0
